service/history/execution/mutable_state_builder.go

```diff
@@ -288,14 +283,13 @@ func NewMutableStateBuilderWithEventV2(
 // NewMutableStateBuilderWithReplicationStateWithEventV2 is used only in test
```

Comment:
nit: is the only difference between v1 and v2 in the persistence Cassandra layer?